src/airflow_declarative/schema.py

```diff
@@ -180,7 +180,7 @@ def represent_callable(self, obj):
         OptionalKey('class'): CLASS,
         OptionalKey('callback'): CLASS | CALLBACK,
         OptionalKey('callback_args'): PARAMS,
-        OptionalKey('args'): OPERATOR_ARGS.allow_extra('*'),
+        OptionalKey('args'): (OPERATOR_ARGS + Dict()).allow_extra('*'),
```

Comment:
`allow_extra` doesn't produce a new immutable instance of a `Dict`, it modifies the original one instead. So `OPERATOR_ARGS.allow_extra('*')` would actually change `OPERATOR_ARGS` Dict rather than create a new instance of it.

This concatenation with an empty Dict produces a new instance, which is being mutated with `allow_extra`, which doesn't change the original `OPERATOR_ARGS` instance.